python-threatexchange/threatexchange/extensions/vpdq/vpdq_index.py

```diff
@@ -26,17 +27,13 @@
 Self = t.TypeVar("Self", bound="VPDQIndex")
 
 
-class VPDQIndexMatch(IndexMatch[IndexT]):
-    def __init__(
-        self,
-        distance: int,
-        query_match_percent: float,
-        compared_match_percent: float,
-        metadata: IndexT,
-    ) -> None:
-        super().__init__(SignalSimilarityInfoWithIntDistance(distance), metadata)
-        self.query_match_percent = query_match_percent
-        self.compared_match_percent = compared_match_percent
+@dataclass
+class VPDQDistance(SignalSimilarityInfo):
```

Comment:
nit: do we want to keep a some what standard naming convention for these objects? e.g. `VPDQSimilarityInfo`